skimage/measure/_marching_cubes.py

```diff
@@ -130,9 +142,6 @@ def marching_cubes(volume, level, spacing=(1., 1., 1.),
     verts = np.asarray(verts)
     faces = np.asarray(faces)
 
-    # Calculate gradient of `volume`, then interpolate to vertices in `verts`
-    grad_x, grad_y, grad_z = np.gradient(volume)
-
```

Comment:
@almarklein You're right - this got left behind in a refactor moving the _correct_mesh_orientation functionality into a separate function.

These days my tools would catch it...